tests/devices/unit_tests/test_bimorph_mirror.py

```diff
@@ -29,117 +30,113 @@ def valid_bimorph_values(mirror: BimorphMirror) -> dict[int, float]:
 
 
 @pytest.fixture
-def mock_vtrgt_vout_propogation(mirror: BimorphMirror):
+def mirror_with_mocked_put(mirror: BimorphMirror):
+    async def busy_idle():
+        await asyncio.sleep(0)
+        set_mock_value(mirror.status, BimorphMirrorStatus.BUSY)
+        await asyncio.sleep(0)
+        set_mock_value(mirror.status, BimorphMirrorStatus.IDLE)
+
+    async def status(*_, **__):
+        asyncio.create_task(busy_idle())
```

Comment:
```suggestion
    async def status(value: Any, wait: bool):
        asyncio.create_task(busy_idle())
```

This should be the correct type annotation and args, I think?

`Any` could probably be `float | BimorphMirrorStatus` but it's fine

Reply:
Added `Any` type hint to both parameters following in-office discussion.